Jantimizer/QueryTestSuite/Program.cs

```diff
@@ -29,14 +30,38 @@ async static Task AsyncMain(string[] args)
             string testBaseDirPath = Path.GetFullPath("../../../Tests");
             DateTime timeStamp = DateTime.Now;
 
-            foreach (DirectoryInfo testDir in new DirectoryInfo(testBaseDirPath).GetDirectories())
-            {
-                TestSuite suite = new TestSuite(connectorSet, timeStamp);
+            if (!File.Exists(Path.Join(testBaseDirPath, "testorder.json")))

```

Comment:
Not too fond of this being in the main-function.

Maybe make a helper that turns the order-list into a list of concrete tests/paths to execute?